class.jetpack.php

```diff
@@ -7124,4 +7133,61 @@ function is_active_and_not_development_mode( $maybe ) {
 		}
 		return true;
 	}
+
+	/* Activated module */
```

Comment:
This file is already huge....could we keep these separate, as they're related with tracking? Perhaps they could be in another class in the tracking package?

Reply:
good idea to not make this class bigger, but this are specific to Jetpack, i.e we don't want to track module activation in VaultPress. let's move them to a file within Jetpack!

Reply:
I like the idea, let's do it 👍

Reply:
Let's follow-up with this in another PR.